core-models/src/main/scala/com/mesosphere/usi/core/models/Ids.scala

```diff
@@ -6,6 +6,16 @@ package com.mesosphere.usi.core.models
   */
 case class PodId(value: String)
 
+/**
+  * Identifier for a pod's task; does not include podId
+  * @param value
+  */
+case class PartialTaskId(value: String)
```

Comment:
What is the relation between a `TaskId` and a `PartialTaskId`?

Reply:
Clarified this by renaming to TaskName.

Also, docs.